aiida/orm/logs.py

```diff
@@ -98,11 +102,38 @@ def delete(self, log_id):
         def delete_many(self, filters):
             """
             Delete all the log entries matching the given filters
+
+            :param filters: filters
+            :type filters: dict
             """
             self._backend.logs.delete_many(filters)
 
     def __init__(self, time, loggername, levelname, dbnode_id, message='', metadata=None, backend=None):  # pylint: disable=too-many-arguments
-        """Construct a new log"""
+        """Construct a new log
+
+        :param time: time
+        :type time: :class:`!datetime.datetime`
+
+        :param loggername: name of logger
+        :type loggername: basestring
+
+        :param levelname: name of log level
+        :type levelname: basestring
+
+        :param dbnode_id: id of database node
+        :type dbnode_id: int
+
+        :param message: log message
+        :type message: basestring
+
+        :param metadata: metadata
+        :type metadata: :class:`!json.json`
```

Comment:
should be just a `dict` no?

Reply:
I put it because of this:
```python
    @property
    def metadata(self):
        """
        Get the metadata corresponding to the entry

        :return: The entry metadata
        :rtype: :class:`!json.json`
        """
        return self._backend_entity.metadata
```
is this incorrect as well?

Reply:
I would say so, it returns a dictionary. I am not even sure what the class `json.json` is?

Reply:
Done!